FORMAT.md

```diff
@@ -74,7 +74,7 @@ The actuator element represents actuators such as the X5-4.  It is assumed to ha
 
 ### `<link>`
 
-The link element refers to a parameterized rigid body.  All links have one output interface.
+The link element refers to a parameterized rigid body.  All links have one output interface.  The `link` element can have no child elements.
```

Comment:
I added explicit language for each element type stating which kinds of children it could have.